### DISCUSSION TITLE
Remember sort order of starred repositories

### DISCUSSION THREAD

**Discussion by CanePlayz** [community/community#45669] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In the starts tab on your profile (i. e. https://github.com/CanePlayz?tab=stars), there is an option to select a sort order in which the repositories should appear:

![image](https://user-images.githubusercontent.com/81419612/215296070-92c8717b-9922-412d-9057-910758e52141.png)

This is really useful, especially if you have starred a lot of repos. However, if you reload that site, they're sorted in the default order again which is really annoying to me. I'd be really useful if GitHub remembered the selected sort order.

Additionally, there could also be a feature added to create a custom sort order, that way accessing the most important reposotories would become eay easier.

**Comment by CameronCSS** [comment]
You can organize them using Lists as Documented [HERE](https://docs.github.com/en/get-started/exploring-projects-on-github/saving-repositories-with-stars#organizing-starred-repositories-with-lists)

Or, an even better option, if you use Chrome there is an extension that will help you. [The FKing Github](https://chrome.google.com/webstore/detail/the-fking-github/agajobpbaphiohkbkjigcalebbfmofdo)

Hope this helps.

**Reply by CanePlayz** [reply]
Yeah, I can do that, unfortunately, I cannot sort them in a custom order there as well ;)

Thanks, I will check out that extension. However, it'd still be cool if it was built in :)
